mk/check.mk

```
@@ -52,7 +52,7 @@ hadolint:
 	find ./tools/releases/dockerfiles/ -type f -iname "Dockerfile*" | grep -v dockerignore | xargs -I {} $(HADOLINT) {}
 
 .PHONY: check
-check: format helm-lint golangci-lint shellcheck kube-lint hadolint ## Dev: Run code checks (go fmt, go vet, ...)
+check: build/ebpf format helm-lint golangci-lint shellcheck kube-lint hadolint ## Dev: Run code checks (go fmt, go vet, ...)
```

format calls generate and generate calls `build/ebpf` so I think this is not needed
```suggestion
check: format helm-lint golangci-lint shellcheck kube-lint hadolint ## Dev: Run code checks (go fmt, go vet, ...)
```

---

you are right

---

addressed